index.html

```diff
@@ -246,6 +246,19 @@ <h3 class="text-colored text-center">Kilo</h3>
 			</div>
 			-->
 
+			<div class="row">
+				<div class="col-md-12 top50">
+					<h3 class="text-colored text-center">Scientific Partner</h3>
+				</div>
+			</div>
+			<div class="row sponsors-kilo">
+				<div class="col-md-12">
+					<a href="http://www.pik-potsdam.de/copan/" class="sponsor" target="_blank">
+						<img src="./hackhpi_assets/2018-partners/pik_new_2.png" alt="Potsdam Institute for Climate Impact Research" style="border: 1px grey solid;">
```

Comment:
Hast du noch ein anderes Bild, als das `pik.png`, was in assets liegt?

Reply:
Sorry, thought I'd pushed the amended commit - should be fixed now.

Reply:
agree, let's keep naming consistent and omit the `_new_2` part 😊